_includes/abstracts/2024/sel4-verification.html

```diff
@@ -0,0 +1,23 @@
+<!--<title>Summit abstract</title>-->
+<!--
+    Copyright 2024, seL4 summit authors (see abstracts)
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+<h4 class="summit-abstract-title">
+    seL4 Verification: Status and Plans
+</h4>
+<p class="summit-abstract-type">
+    Talk
+</p>
+<p class="summit-abstract-author">
+    Presented by Gerwin Klein, <span class="summit-abstract-affiliation">Proofcraft</span>
```

Comment:
This will not be presented by Gerwin. We're still planning on it, for now please have the full list of authors. Thanks :)